modules/olmv1-cluster-extension-permissions.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-cluster-extension-permissions_{context}"]
+= Cluster extension permissions
+
+{olmv1-first} is designed to be secure by default. Unlike {olmv0}, {olmv1} does not have cluster administrator permissions. As a result, cluster administrators must create a service account for each cluster extension and assign the role-based access controls (RBAC) required to install and manage that extension.
```

Comment:
wdyt if we rephrase it to be something like:

```suggestion
{olmv1-first} is designed to be secure by default. Rather than managing all cluster extensions with a single cluster administrator scoped service account, as in {olmv0}, {olmv1} uses manages each cluster extension as the service account specified in its CR. As a result, cluster administrators can create a service account for each cluster extension and assign role-based access controls (RBAC) with only the permissions required to install and manage that extension.
```
(cc @joelanford)

Reply:
I took a stab at incorporating your feedback here: https://github.com/openshift/openshift-docs/pull/87627/commits/fae63c316ff38806258ab3a372ab41e101ca35b8#diff-6bd37e1983027f833fe453f5082e41498125ca7d4e045760fd40575eee8b84ddR10

WDYT?